### PR TITLE
[com_actionlogs] - fix for plugin link

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #23841

### Summary of Changes
now  the plugin link open the plugin edit

### Testing Instructions
see #23841

### CODE REVIEW

File: administrator/components/com_actionlogs/helpers/actionlogs.php [hunk 1]
```diff
@@ -245,9 +245,9 @@ public static function getContentTypeLink($component, $contentType, $id, $urlVar
 			}
 		}
 
-		if (empty($urlVar))
+		if ($component==='com_plugins')
```
Comment (Quy) [inline administrator/components/com_actionlogs/helpers/actionlogs.php]:
```suggestion
		if ($component === 'com_plugins')
```

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on 00a464f97865b60db5adfe2b707f428a6ee973bc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23853">issues.joomla.org/tracker/joomla-cms/23853</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 00a464f97865b60db5adfe2b707f428a6ee973bc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23853">issues.joomla.org/tracker/joomla-cms/23853</a>.</sub>

**Comment by brianteeman** [comment]
Thanks @alikon

**Comment by joomdonation** [comment]
Sorry but this does not look like a right fix to me. We should not introduce component specific logic into that method. There are better ways to fix this:

1. The most safe method is adding new language items for this specific plugin content type, 4 new language items needed (we use similar method for plugin installed/uninstalled already https://github.com/joomla/joomla-cms/blob/staging/administrator/language/en-GB/en-GB.plg_actionlog_joomla.ini#L53. Something like:

```text
PLG_ACTIONLOG_JOOMLA_PLUGIN_PUBLISHED="User <a href='{accountlink}'>{username}</a> published the plugin <a href='index.php?option=com_plugins&task=plugin.edit&extension_id={id}'>{title}</a>"
PLG_ACTIONLOG_JOOMLA_PLUGIN_UNPUBLISHED="User <a href='{accountlink}'>{username}</a> umpublished the plugin <a href='index.php?option=com_plugins&task=plugin.edit&extension_id={id}'>{title}</a>"
PLG_ACTIONLOG_JOOMLA_PLUGIN_ARCHIVED="User <a href='{accountlink}'>{username}</a> ARCHIVED the plugin <a href='index.php?option=com_plugins&task=plugin.edit&extension_id={id}'>{title}</a>"
PLG_ACTIONLOG_JOOMLA_PLUGIN_TRASHED="User <a href='{accountlink}'>{username}</a> trashed the plugin <a href='index.php?option=com_plugins&task=plugin.edit&extension_id={id}'>{title}</a>"
```

2. If we cannot introduce new language items now because we are in RC state, then you can add **getContentTypeLink** method to PluginsHelper class https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_plugins/helpers/plugins.php to return the link in expected format

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 00a464f97865b60db5adfe2b707f428a6ee973bc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23853">issues.joomla.org/tracker/joomla-cms/23853</a>.</sub>

**Comment by SharkyKZ** [comment]
See #23857 for alternative please.

**Comment by HLeithner** [comment]
#23857 seams to be the correct way to do it, please test it so I can merge it before release get packaged tomorrow.

**Pull request closed without merging**